sunpy/database/tests/test_tables.py

```diff
@@ -509,8 +509,8 @@ def test_create_display_table():
         'observation_time_start', 'instrument', 'size',
         'wavemin', 'path', 'starred', 'tags']
     table = _create_display_table(entries, columns)
-    filedir = os.path.dirname(os.path.realpath(__file__))
-    with open(os.path.join(filedir, 'test_table.txt'), 'r') as f:
+    filedir = Path(Path(__file__).resolve()).parent
+    with open(str(Path.home().joinpath(filedir, 'test_table.txt')), 'r') as f:
```

Comment:
can `open` not work on path objects?

Reply:
as in do you want open to have os.path in it instead of Path objects?

Reply:
No. 

Does `with open(Path.home().joinpath(filedir, 'test_table.txt'), 'r')` not work?

Reply:
`open` accepts `Path` objects on 3.6+ (at least in my experience)

Reply:
yeah i made those changes now